cmd/peer-rest-client.go

```diff
@@ -467,6 +467,38 @@ func (client *peerRESTClient) BackgroundHealStatus() (madmin.BgHealState, error)
 	return state, err
 }
 
+// BgHealingOpsStatus describes the status
+// of the auto healing operations
+type BgHealingOpsStatus struct {
+	LastActivity time.Time
+}
+
+// BgLifecycleOpsStatus describes the status
+// of the background lifecycle operations
+type BgLifecycleOpsStatus struct {
+	LastActivity time.Time
+}
+
+// BgOpsStatus describes the status of all
+// operations performed in background such
+// as auto-healing and lifecycle.
+type BgOpsStatus struct {
+	HealingOps   BgHealingOpsStatus
+	LifecycleOps BgLifecycleOpsStatus
+}
+
+func (client *peerRESTClient) BackgroundOpsStatus() (BgOpsStatus, error) {
```

Comment:
Would `BgOpsStatus` data type be suitable for expansion as we add more background operations in the future?

Reply:
> Would `BgOpsStatus` data type be suitable for expansion as we add more background operations in the future?

yes, BgOpsStatus is intended to hold all background information in the future, that's why it has a generic name